compliance-report/action.yml

```diff
@@ -1,9 +1,6 @@
 name: Generate a compliance report
 description: Generates the compliance report in the S3_ASSETS folder
 inputs:
-  token:
```

Comment:
Did you mean to delete the entire input or only the `required` option?

Reply:
It is moved to the bottom

Reply:
🤦